### PR TITLE
Added tests for validators

### PR DESCRIPTION
**Pull request by skramar**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=738)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [x] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/tests/unittests/validators/testmodel.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+"""Validators Test."""
```
Comment (makarman) [inline eventually/tests/unittests/validators/testmodel.py]: Why validators/testmodel.py?. We don't test models in validators. Maybe better validators/test.py or validators/testfunctions.py

File: eventually/tests/unittests/validators/testmodel.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+from django.test import TestCase
+from utils.validators import duration_validator, timestamp_validator, required_keys_validator
+
+
+class CommentModelTestCase(TestCase):
```
Comment (makarman) [inline eventually/tests/unittests/validators/testmodel.py]: Why CommentModelTestCase?

File: eventually/tests/unittests/validators/testmodel.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    """TestCase for providing validators testing."""
+
+    def test__duration_validator_success(self):
```
Comment (makarman) [inline eventually/tests/unittests/validators/testmodel.py]: Why double '__' in test__duration_validator_success?

File: eventually/tests/unittests/validators/testmodel.py (hunk 1 above)
Comment (skramar) [reply eventually/tests/unittests/validators/testmodel.py]: done

File: eventually/tests/unittests/validators/testmodel.py (hunk 2 above)
Comment (skramar) [reply eventually/tests/unittests/validators/testmodel.py]: done

File: eventually/tests/unittests/validators/testmodel.py (hunk 3 above)
Comment (skramar) [reply eventually/tests/unittests/validators/testmodel.py]: done

**Review by makarman — DISMISSED** [review]

File: eventually/tests/unittests/validators/testvalidators.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
+"""Validators Test."""
```
Comment (redmyt) [inline eventually/tests/unittests/validators/testvalidators.py]: Please emphasize the name of the module with ====... chars. It makes it more readable with sphinx docs =)

File: eventually/tests/unittests/validators/testvalidators.py [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+        self.assertTrue(is_valid)
+
+    def test_duration_validator_fail_string(self):
+        """Method that tests `duration_validator`."""
+        value = "some_text"
+
+        is_valid = duration_validator(value)
+
+        self.assertFalse(is_valid)
```
Comment (redmyt) [inline eventually/tests/unittests/validators/testvalidators.py]: `assertIsNone` not `assertFalse`

File: eventually/tests/unittests/validators/testvalidators.py [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+    def test_duration_validator_fail_not_positive(self):
+        """Method that tests `duration_validator`."""
+        value = -25
+
+        is_valid = duration_validator(value)
+
+        self.assertFalse(is_valid)
```
Comment (redmyt) [inline eventually/tests/unittests/validators/testvalidators.py]: `assertIsNone` not `assertFalse`

File: eventually/tests/unittests/validators/testvalidators.py [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+    def test_duration_validator_fail_overflow(self):
+        """Method that tests `duration_validator`."""
+        value = 250000000000000000
+
+        is_valid = duration_validator(value)
+
+        self.assertFalse(is_valid)
```
Comment (redmyt) [inline eventually/tests/unittests/validators/testvalidators.py]: `assertIsNone` not `assertFalse`

File: eventually/tests/unittests/validators/testvalidators.py [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+        self.assertTrue(is_valid)
+
+    def test_timestamp_validator_fail(self):
+        """Method that tests `timestamp_validator`."""
+        value = 'some_text'
+
+        is_valid = timestamp_validator(value)
+
+        self.assertFalse(is_valid)
```
Comment (redmyt) [inline eventually/tests/unittests/validators/testvalidators.py]: `assertIsNone` not `assertFalse`

**Review by ssOleg — DISMISSED** [review]

**Review by ssOleg — APPROVED** [review]

**Pull request merged**